src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) == null) {
+            return false;
+        } else if (userService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
try to rewrite this `if-elseif-else` into 1 line using &&

return ...checkIfUserIsNotNull... && ...checkIfPasswordsAreEqual...;